### PR TITLE
Only set ipv6 killswitch if ipv6 interface exists

### PR DESCRIPTION
**Pull request by bobbypage**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -17,9 +17,12 @@ docker_network="$(ip -o addr show dev eth0 | awk '$3 == "inet" {print $4}')"
 docker_network_rule=$([ ! -z "$docker_network" ] && echo "! -d $docker_network" || echo "")
 iptables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker_network_rule -j REJECT
 
-docker6_network="$(ip -o addr show dev eth0 | awk '$3 == "inet6" {print $4}')"
-docker6_network_rule=$([ ! -z "$docker6_network" ] && echo "! -d $docker6_network" || echo "")
-ip6tables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker6_network_rule -j REJECT
+if [[ -z "${docker6_network}" ]]; then
```
Comment (jordanpotter) [inline entrypoint.sh:20]: Won't this condition always evaluate to `true`? I think we're missing the line where we set the `docker6-network` variable:
```suggestion
docker6_network="$(ip -o addr show dev eth0 | awk '$3 == "inet6" {print $4}')"
if [[ -z "${docker6_network}" ]]; then
```

Comment (bobbypage) [reply entrypoint.sh:21]: Oops, you’re completely correct, copy and paste error from the other PR. Thanks for the catch!

File: entrypoint.sh [hunk 2]
```diff
@@ -18,8 +18,12 @@ docker_network_rule=$([ ! -z "$docker_network" ] && echo "! -d $docker_network"
 iptables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker_network_rule -j REJECT
 
 docker6_network="$(ip -o addr show dev eth0 | awk '$3 == "inet6" {print $4}')"
-docker6_network_rule=$([ ! -z "$docker6_network" ] && echo "! -d $docker6_network" || echo "")
-ip6tables -I OUTPUT ! -o $interface -m mark ! --mark $(wg show $interface fwmark) -m addrtype ! --dst-type LOCAL $docker6_network_rule -j REJECT
+if [[ -z "${docker6_network}" ]]; then
```
Comment (jordanpotter) [inline entrypoint.sh:21]: Hrmm, judging from line 6, should this line just be:
```
if [[ -z $docker6_network ]]; then
```

**Comment by jordanpotter** [comment]
From https://github.com/jordanpotter/docker-wireguard/pull/3#discussion_r487465459, it sounds like this change was required because your Docker setup doesn't have ipv6 support. Strangely, mine doesn't have ipv6 configured either, but I never needed this change 🤔 

Would you mind including the error you saw?

File: entrypoint.sh (hunk 2 above)
Comment (bobbypage) [reply entrypoint.sh:21]: hmm, is there any difference other than the quoting? I just quoted as out of habit to prevent shell splitting/expansion.

**Comment by bobbypage** [comment]
This was the error I was hitting without this change:

```
$ sudo docker run --name wireguard --cap-add NET_ADMIN --cap-add SYS_MODULE --sysctl net.ipv4.conf.all.src_valid_mark=1 -v $(pwd)/config/mullvad/mullvad-ca10-ipv4.conf:/etc/wireguard/mullvad.conf bobbypage/docker-wireguard:latest
[#] ip link add mullvad type wireguard
[#] wg setconf mullvad /dev/fd/63
[#] ip -4 address add<redacted> dev mullvad
[#] ip link set mtu 1420 up dev mullvad
[#] resolvconf -a mullvad -m 0 -x
[#] wg set mullvad fwmark 51820
[#] ip -4 route add 0.0.0.0/0 dev mullvad table 51820
[#] ip -4 rule add not fwmark 51820 table 51820
[#] ip -4 rule add table main suppress_prefixlength 0
skipping setting net.ipv4.conf.all.src_valid_mark
[#] echo skipping setting net.ipv4.conf.all.src_valid_mark
[#] iptables-restore -n
ip6tables v1.8.4 (legacy): Couldn't load match `addrtype':No such file or directory
```

File: entrypoint.sh (hunk 2 above)
Comment (jordanpotter) [reply entrypoint.sh:21]: I'm cool either way, as long as the two lines (6 and 21) are consistent 👍

Comment (bobbypage) [reply entrypoint.sh]: Sounds good, updated to keep them consistent.

**Review by jordanpotter — APPROVED** [review]
LGTM!

**Comment by jordanpotter** [comment]
Thanks again @bobbypage!!

**Pull request merged**